constraints.pro

```diff
@@ -20,7 +20,7 @@ gen_enforced_dependency(WorkspaceCwd, DependencyIdent, DependencyRange2, Depende
   % A list of exception to same version rule
   \+ member(DependencyIdent, [
     % Allow enzyme example workspace use a older version react and react-dom, because enzyme don't support react 17
-    'react', 'react-dom',
+    'react', 'react-dom', '@types/react',
```

Comment:
Is it ok to add an exception for React types too considering React is already in the exceptions? Looks fine to me

Reply:
yeah, makes sense